src/workloads/execution/TimeSeries2dsphere.yml

```diff
@@ -0,0 +1,145 @@
+  documentCount: &documentCount 1036800
+  # Since we have a relatively small amount of data, we don't want to use an area that's too
+  # big (such as the entire Earth, or a whole continent). That would make the data very
+  # sparse over that area. These values roughly correspond to New York State, which is
+  # hopefully a good scale.
+  minX: &minX -80.0
+  maxX: &maxX -73.0
+  minY: &minY 40.0
+  maxY: &maxY 45.0
+  # Suppose each sensor moves at around 60 mph or 100 kph.
+  # This comes out to around 30 m/s.
+  # Events are 10 seconds apart, so that's 300m per event.
+  # 90 degrees of longitude or latitude is roughly 10,000 km = 10,000,000m.
+  # So (300m / event) * (90 deg / 10,000,000m) = 0.0027 deg / event.
+  # This is very rough, and doesn't account for how 1 degree longitude has a different length
+  # at different latitudes. The idea is just to get roughly the right scale.
+  # Another check: 0.0027 degrees per doc and 1,000 docs per series means 2.7 degrees per
+  # series, which is roughly the scale of our min/max X/Y.
+  distPerDoc: &distPerDoc 0.0027
+
+  # Other abbreviations.
+  nop: &Nop {Nop: true}
+
+
+Actors:
+- Name: CreateTimeSeriesCollection
+  Type: RunCommand
+  Threads: 1
+  Phases:
+  - Repeat: 1
+    Database: *db
+    Operation:
+      OperationMetricsName: CreateTimeSeriesCollection
+      OperationName: RunCommand
+      OperationCommand:
+        # We have to call the collection 'Collection0' because that's the name that Loader
+        # chooses implicitly.
+        {create: &coll Collection0, timeseries: {timeField: "timestamp", metaField: "meta"}}
+  - *Nop
+  - *Nop
+  - *Nop
+
+- Name: InsertData
+  Type: Loader
+  Threads: 1
+  Phases:
+  - *Nop
+  - Repeat: 1
+    Database: *db
+    Threads: 1
+    CollectionCount: 1
+    DocumentCount: *documentCount
+    BatchSize: *batchSize
+    Document:
+      # start "2022-01-01" = "2022-01-01T00:00:00Z", step 10000 = size of step 10 seconds or 10,000 millis
+      # Note that the timestamp does not restart when we change from one series to the next,
+      # so that's unrealistic, but it shouldn't matter for this test because the query and
+      # the index both ignore the time field.
+      timestamp: {^IncDate: {start: "2022-01-01", step: 40000}}
+      location: {^TwoDWalk: {
+        docsPerSeries: *docsPerSeries,
+        minX: *minX,
+        maxX: *maxX,
+        minY: *minY,
+        maxY: *maxY,
+        distPerDoc: *distPerDoc,
+      }}
+    Indexes:
+    - *index
+  - *Nop
+  - *Nop
+
+- Name: Quiesce
+  Type: QuiesceActor
+  Threads: 1
+  Database: *db
+  Phases:
+  - *Nop
+  - *Nop
+  - Repeat: 1
+  - *Nop
+
+- Name: GeoQueries
+  Type: RunCommand
+  Threads: 1
+  Phases:
+  - *Nop
+  - *Nop
+  - *Nop
+  - Duration: 10 seconds  # TODO more
+    Database: *db
+    Operations:
+    - OperationMetricsName: GeoWithinQuery
+      OperationName: RunCommand
+      OperationCommand:
+        aggregate: Collection0
+        pipeline: [
```

Comment:
From what I can tell, this pipeline doesn't match any results right now because of the 2dsphere index bug. We could wait for the bug fix to merge, or just merge this now and check that its matching results once the bug fix merges. I think worst case we'll get a BF when the bug fix merges but not sure